src/Data/ProgrammesDb/Entity/RefOptions.php

```diff
@@ -23,147 +27,112 @@ class RefOptions
     private $id;
 
     /**
-     * @var array
+     * @var string
      *
-     * @ORM\Column(type="string", nullable=false)
+     * @ORM\Column(type="string", length=36, unique=true, nullable=false)
      */
-    private $originalId;
+    private $guid;
 
     /**
-     * @var CoreEntity
+     * @var string
      *
-     * @ORM\ManyToOne(targetEntity="CoreEntity")
-     * @ORM\JoinColumn(nullable=true, onDelete="SET NULL")
+     * @ORM\Column(type="string", length=35, nullable=false)
      */
-    private $optionsForCoreEntity;
+    private $projectId;
 
     /**
-     * @var Network
+     * @var integer
      *
-     * @ORM\ManyToOne(targetEntity="Network")
-     * @ORM\JoinColumn(nullable=true, onDelete="SET NULL")
-     */
-    private $optionsForNetwork;
-
-    /**
-     * @var array
-     *
-     * @ORM\Column(type="json_array", nullable=true)
+     * @ORM\Column(type="integer", nullable=false)
```

Comment:
entityId would be pid or network ID that comes from isite so it should be a string